x-pack/test/api_integration/apis/upgrade_assistant/es_deprecation_logs.helpers.ts

```diff
@@ -30,7 +30,6 @@ const deprecationMock = {
   message:
     '[types removal] Specifying include_type_name in get index template requests is deprecated.',
   'data_stream.type': 'logs',
-  'data_stream.dataset': 'deprecation.elasticsearch',
```

Comment:
To be an accurate mock we should rather change the value to elasticsearch.deprecation instead of removing the field completely